src/components/participant-image/participant-image.tsx

```diff
@@ -0,0 +1,32 @@
+import { PropsWithChildren } from "@/types/react";
```

Comment:
It's getting better. You can move this component (and the others related to participants list) to components/participants-list. Since we're using it as "compound components" we could use colocation to organize them.
See the sidebar component as an example.

Reply:
@vmarcosp It's a good idea. Should I keep the folder prefix? example "participant-name, participant-image"

Reply:
Yep and you can do something like:
```
src/
  components/
    participants/
      participants-list.tsx
      participant-item.tsx
      participant-image.tsx
      participant-name.tsx
```

Reply:
Done